pkg/crc/preflight/preflight_checks_tray_windows.go

```diff
@@ -19,130 +17,119 @@ import (
 )
 
 func checkIfTrayInstalled() error {
-	/* We want to force installation whenever setup is ran
-	 * as we want the service config to point to the executable
-	 * with which the setup command was issued.
-	 */
-
-	return fmt.Errorf("Ignoring check and forcing installation of System Tray")
+	if os.FileExists(filepath.Join(constants.StartUpFolder, constants.TrayShortcutName)) {
+		return nil
+	}
+	return fmt.Errorf("CodeReady Containers tray is not Installed")
 }
 
-func fixTrayInstalled() error {
-	/* To avoid asking for elevated privileges again and again
-	 * this takes care of all the steps needed to have a running
-	 * tray by invoking a ps script which does the following:
-	 * a) Add logon as service permission for the user
-	 * b) Create the daemon service and start it
-	 * c) Add tray to startup folder and start it
-	 */
+func checkIfDaemonInstalled() error {
+	// We want to force the installation of daemon as we want to
+	// run the daemon from the crc executable which ran the setup
+	return fmt.Errorf("Ignoring check, forcing installation of daemon")
```

Comment:
will that cause issue if one runs `crc setup` when the daemon is already running?

Reply:
This was causing issue, now during the fix we call `stopDaemon` which makes sure that the socket is usable and the issue is fixed.